setup.cfg

```diff
@@ -1,51 +1,74 @@
-[build_sphinx]
-source-dir = docs
-build-dir = docs/_build
-all_files = 1
+[metadata]
+name = sunkit_dem
+author = SunPy Developers
+author_email = 
```

Comment:
@wtbarnes - did you want to put your email address here?

Reply:
```suggestion
author_email = will.t.barnes@gmail.com
```

Reply:
Sure 😄